app/components/common/blank_results_component.rb

```diff
@@ -0,0 +1,20 @@
+module Common
+  class BlankResultsComponent < ViewComponent::Base
+    def initialize(reason = :not_found)
+      case reason
+      when :empty
+        @text1 = "Žiadne záznamy"
```

Comment:
Toto je strasne genericke, blank stav by ti mal aj trochu poradit, ze naco to sluzi (napr. filtre) a ponuknut akciu (napr. vytvorit filter)

Reply:
Je to teda v prekladoch. Ak vies ako, kludne pomen